modules/ols-custom-resource-definition-file-parameters.adoc

```diff
@@ -0,0 +1,498 @@
+    providers:
+      deploymentName:
+|Name of the {azure-openai} deployment.
+|string
+|No
+
+|spec:
+  llm:
+    providers:
+      projectID:
+|The {watsonx} project ID.
+|string
+|No
+
+|spec:
+  llm:
+    providers:
+      credentialsSecretRef:
+|The name of the secret object that stores the API provider credentials
+|string.
+|No
+
+|spec:
+  llm:
+    providers:
+      models:
+|List of models provided by the Large Language Model (LLM) provider.
+|string
+|No
+|===
+
+.{ols-short} CRD parameters for Large Language Model (LLM) provider models
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  llm:
+    providers:
+      models:
+        name:
+|Model name.
+|string
+|yes
+
+|spec:
+  llm:
+    providers:
+      models:
+        url:
+|URL for the API model. The URL must start with `https://`.
+|URL
+|No
+
+|spec:
+  llm:
+    providers:
+      models:
+        contextWindowSize:
+|Defines the size of the context window that the LLM model uses for conversations. The default setting is specific to each LLM provider model. Verify the size of the context window for the specific model you are using. The minimum value that this parameter can accept is `1024`. If this parameter is not set, the default value is `8192`.
+|Integer
+|No
+
+|spec:
+  llm:
+    providers:
+      models:
+        parameters:
+|API parameters for the model. Most parameters are unique for each model.
+|Dictionary
+|No
+|===
+
+.{ols-short} CRD parameters for Large Language Model (LLM) provider model parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  llm:
+    providers:
+      maxTokensForResponse:
+|Maximum number of response tokens that the LLM provider uses. If this parameter is not set, the default value is `1024`.
+|Integer
+|No
+|===
+
+.{ols-short} CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  ols:
+    defaultModel:
+|The name of the default model.
+|String
+|Yes
+
+|spec:
+  ols:
+    defaultProvider:
+|Provider that the model uses by default.
+|String
+|No
+
+|spec:
+  ols:
+    maxMemory:
+|Redis maxmemory value.
+|Integer or string. For example, `1024mb`
+|No
+
+|spec:
+  ols:
+    maxMemoryPolicy:
+|Redis `maxmemory` policy. The default value is `allkeys-lru`.
+|String
+|No
+
+|spec:
+  ols:
+    logLevel:
+|The log level. The default is `Info`
+|DEBUG,INFO, WARNING, ERROR, CRITICAL
+|No
+
+|spec:
+  ols:
+    queryFilters:
+|Contains the list of query filters.
+|Array or string.
+|No
+|===
+
+.{ols-short} user data collection CRD parameters
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  ols:
+    userDataCollection:
+|User data collection switches.
+|Object
+|No
+
+|spec:
+  ols:
+    userDataCollection:
+      feedbackDisabled:
+|If you set the parameter to `1` the container does not send user feedback to Red{nbsp}Hat.
```

Comment:
spec:ols: userDataCollection:   - We Dont need this block

Reply:
removed the section